test/unit/spec/common/cellComponents/paramsWidget-Spec.js

```diff
@@ -70,32 +69,43 @@ define([
                 await this.paramsWidgetInstance.stop();
             }
             window.kbaseRuntime = null;
-            this.parent.innerHTML = '';
+            this.parent.remove();
         });
 
-        it('should render the correct parameters', () => {
+        it('should render the correct parameters', function () {
             //Regular (non-advanced) params
-            const paramContainers = $('div.kb-field-cell__param_container');
+            const paramContainers = this.node.querySelectorAll(
+                'div.kb-field-cell__param_container'
+            );
+            const titleList = Object.values(this.parameters.specs).map((item) => {
+                return item.ui.label;
+            });
 
-            paramContainers.each(function () {
+            paramContainers.forEach((paramNode) => {
                 //each param container should have ONE label
-                const label = $(this).find('label.kb-field-cell__cell_label');
+                const labelArr = paramNode.querySelectorAll('label.kb-field-cell__cell_label');
+                expect(labelArr.length).toBe(1);
+                const label = labelArr[0];
                 expect(label).toBeDefined();
-                expect(label.length).toBe(1);
+                expect(titleList.includes(label.innerText)).toBeTrue();
```

Comment:
add in extra tests here - the label and the title should be from the parameter specs